lecture10/iam.yml

```
@@ -0,0 +1,31 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Resources: 
+  IamRoletoAccessS3:

```

パスカルケースにするならtoも区切り文字になるので`t`は大文字にしましょう！
```suggestion
  IamRoleToAccessS3:
```

---

toとforの`t`と`f`を大文字にしました。